core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -162,8 +161,11 @@ public AccessToken refreshAccessToken() throws IOException {
         }
       };
     }
-
-    throw new RuntimeException("Not supporting credentials of type " + source.getClass().getName());
+    throw new RuntimeException(
+        String.format(
+            "[%s] Unable to connect via automatic IAM authentication: "
+                + "Not supporting credentials of type %s",
```

Comment:
Should this be "Unsupported credentials of type ..."? "Not supporting" sounds odd.

Reply:
fixed.